Policies/github-terms-of-service.md

```diff
@@ -161,7 +161,7 @@ To the extent such an agreement is not enforceable by applicable law, you grant
 Paid accounts may have [private repositories](/articles/making-a-public-repository-private/), which allow the User to control access to Content. GitHub employees [only access private repositories when access is required for security or maintenance](/articles/github-security/#employee-access) or for support reasons, and then only with the consent of the repository owner. Because we respect your privacy in private repositories, you must respect the Terms of Service in everything you post to a private repository. If we have reason to believe the contents of a private repository are in violation of the law or of these Terms, we have the right to remove them.
 
 ### E. Copyright Infringement and DMCA Policy
-If you believe that content on our website violates your copyright, please contact us in accordance with our [Digital Millennium Copyright Act Policy](/articles/dmca-takedown-policy/). If you are a copyright owner and you believe that content on GitHub violates your rights, please contact us via [our convenient DMCA form](https://github.com/contact/dmca) or by emailing copyright@github.com. There may be legal consequences for sending a false or frivolous takedown notice. Before sending a takedown request, you must consider legal uses such as fair use and licensed uses.
+If you believe that content on our website violates your copyright, please contact us in accordance with our [Digital Millennium Copyright Act Policy](/articles/dmca-takedown-policy/). If you are a copyright owner and you believe that content on GitHub violates your rights, please contact us via [our convenient DMCA form](https://github.com/contact/dmca) or by emailing copyright@github.com. There may be legal consequences for sending a false or frivolous takedown notice. Before sending a take-down request, you must consider legal uses such as fair use and licensed uses.
```

Comment:
I agree that usage should be consistent for the word. Inconsistency looks unprofessional and in legal land could throw a user into a state of confusion, making them wonder whether the sleeping difference implies something slightly different. And until they figure it the difference or reconcile the problem, they may be deterred from using GitHub. Would you mind making a PR indoor just this one fix and cite simply that consistency would be less confusing to users? Unless of course I missed something in this thread. Just because they disagreed with other changes does not mean they disagreed with this particular change.